<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWindow</class>
 <widget class="QMainWindow" name="ImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="imagelabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Convert_to">
     <property name="title">
      <string>&amp;Convert to</string>
     </property>
     <addaction name="action1_bit"/>
     <addaction name="action8_bit"/>
     <addaction name="action32_bit"/>
    </widget>
    <addaction name="actionFlip_Horizontally"/>
    <addaction name="actionFlip_Vertically"/>
    <addaction name="action_Resize"/>
    <addaction name="separator"/>
    <addaction name="menu_Convert_to"/>
    <addaction name="actionNiuqu"/>
    <addaction name="actionGray"/>
    <addaction name="actionRUIHUA"/>
    <addaction name="actionPINGHUA"/>
    <addaction name="actionSOBEL"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>Scan</string>
    </property>
    <addaction name="actionHd"/>
    <addaction name="actionFile"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
   <addaction name="menuScan"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFlip_Horizontally">
   <property name="text">
    <string>Flip &amp;Horizontally</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFlip_Vertically">
   <property name="text">
    <string>Flip &amp;Vertically</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Resize">
   <property name="text">
    <string>&amp;Resize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action1_bit">
   <property name="text">
    <string>1 bit</string>
   </property>
  </action>
  <action name="action8_bit">
   <property name="text">
    <string>8 bit</string>
   </property>
  </action>
  <action name="action32_bit">
   <property name="text">
    <string>32 bit</string>
   </property>
  </action>
  <action name="actionHd">
   <property name="text">
    <string>Slideshow</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>Openfile</string>
   </property>
  </action>
  <action name="actionNiuqu">
   <property name="text">
    <string>Gossian</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionRUIHUA">
   <property name="text">
    <string>RUIHUA</string>
   </property>
  </action>
  <action name="actionPINGHUA">
   <property name="text">
    <string>PINGHUA</string>
   </property>
  </action>
  <action name="actionSOBEL">
   <property name="text">
    <string>SOBEL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
